src/components/Spacing/Spacing.story.tsx

```diff
@@ -15,12 +15,15 @@
 
 import React, { Fragment } from 'react';
 
-import Spacing from './Spacing';
+import { Spacing } from './Spacing';
 import Button from '../Button';
 
 export default {
   title: 'Layout/Spacing',
-  component: Spacing
+  component: Spacing,
+  parameters: {
+    jest: ['Spacing']
+  }
```

Comment:
I removed the Storybook Jest addon in #621, so these lines are obsolete. Can you remove them, please?